corehq/apps/data_dictionary/util.py

```diff
@@ -62,16 +55,32 @@ def _get_current_case_types_and_properties(domain):
 
 
 def add_properties_to_data_dictionary(domain, case_type, properties):
-    case_type_obj, created = CaseType.objects.get_or_create(domain=domain, name=case_type)
-    if not created:
-        old_properties = set(CaseProperty.objects.filter(case_type=case_type_obj).values_list('name', flat=True))
-    else:
-        old_properties = set()
-    new_properties = []
-
-    for case_property in properties:
-        if case_property not in old_properties:
-            old_properties.add(case_property)
-            new_properties.append(CaseProperty(name=case_property, case_type=case_type_obj))
-
-    CaseProperty.objects.bulk_create(new_properties)
+    if not properties:
```

Comment:
This seems like a strange way of writing this, since the caller isn't expecting this function to return anything. It might be clearer if you do
```py
if properties:
    _create_properties_for_case_types(...)
```